plugins/woocommerce/client/admin/client/customize-store/style.scss

```
@@ -32,292 +32,283 @@
-	.woocommerce-customize-store__theme-switch-warning-modal-footer {
-		display: flex;
-		gap: 12px;
-		justify-content: flex-end;
+	.woocommerce-edit-site-layout__header-container {
+		position: static;
+
+		@media screen and (max-width: 782px) {
+			display: none;
+		}
 	}
 }
 
-.woocommerce-customize-store__opt-in-usage-tracking-modal {
-	width: 480px;
-	max-width: 480px;
+.woocommerce-customize-store-main {
+	flex: 1;
+	padding: 2rem 2.5rem;
+	overflow: auto;
 
-	a {
-		text-decoration: underline;
-		color: inherit;
-	}
-
-	.components-modal__header {
-		padding: 32px 32px 16px 32px;
+	@media screen and (max-width: 782px) {
+		padding: 1rem;
+		width: 100%;
 	}
 
-	.components-modal__content {
-		padding: 0;
-	}
+	.woocommerce-customize-store-banner {
+		background: rgba(242, 237, 255, 0.6);
+		border-radius: 4px;
+		display: flex;
+		margin-bottom: 1.5rem;
+		min-height: 248px;
+		width: 100%;
+		align-items: center;
+		padding: 2rem 3rem;
+		position: relative;
+		overflow: hidden;
 
-	.core-profiler__checkbox {
-		padding: 16px 32px 16px 32px;
+		@media screen and (max-width: 782px) {
+			padding: 1.5rem;
+			min-height: 200px;
+		}
 
-		div {
+		.woocommerce-banner-visual {
+			position: absolute;
+			right: 0;
+			top: 50%;
+			transform: translateY(-50%);
 			display: flex;
+			align-items: center;
+			justify-content: center;
+			height: 100%;
+			pointer-events: none;
 
-			.components-checkbox-control__input-container {
-				display: block;
+			@media screen and (max-width: 1024px) {
+				display: none;
 			}
-		}
-	}
 
-	.woocommerce-customize-store__design-change-warning-modal-footer {
-		border-top: 1px solid #ddd;
-		padding: 24px 32px 32px 32px;
-		width: 100%;
-		display: flex;
-		gap: 12px;
-		justify-content: flex-end;
+			.woocommerce-banner-shape {
+				position: absolute;
+			}
 
-		button.is-link {
-			padding: 6px 12px;
-			text-decoration: none;
+			.woocommerce-banner-icon {
+				position: relative;
+			}
 		}
-	}
 
-}
+		.woocommerce-customize-store-banner-content {
+			width: 100%;
+			max-width: 60%;
 
-.woocommerce-onboarding-loader {
-	min-height: 100vh;
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	flex-direction: column;
-	background-color: #fff;
+			@media screen and (max-width: 1024px) {
+				max-width: 100%;
+			}
 
-	.woocommerce-onboarding-loader-wrapper {
-		align-items: flex-start;
-		display: flex;
-		flex-direction: column;
-		justify-content: center;
-		max-width: 520px;
-		min-height: 400px;
+			button.components-button {
+				padding: 8px 16px;
+				height: 40px;
+			}
 
-		.woocommerce-onboarding-loader-container {
-			text-align: center;
-			min-height: 400px;
-		}
+			h1,
+			h2 {
+				font-size: 1.25rem;
+				line-height: 23.87px;
+				font-weight: 500;
+				color: #000;
+				padding: 0;
+			}
 
-		.loader-hearticon {
-			position: relative;
-			top: 2px;
-			left: 2px;
-		}
-	}
-}
+			p {
+				margin: 12px 0 28px 0;
+				color: #2f2f2f;
+				line-height: 16px;
```

Very nitpicky, but where do these numbers come from? In general, I think it's a good practice not to override `line-height` if it's not really necessary, so the text inherits the line height of the rest of the app. For margins, I wonder if we could use `em` instead of px?


```suggestion
				margin: 1em 0 2em 0;
				color: #2f2f2f;
```

---

Styles are reused from CYS banners `plugins/woocommerce/client/admin/client/customize-store/intro/intro.scss` e.g. [here](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/client/admin/client/customize-store/intro/intro.scss#L212). I didn't analyse the styles per se and I'm a bit reluctant of changing them in scope of this PR. For example classes `banner-actions` or `mobile-header` were also in CYS before and are still in use in LYS.